assets/queries/cloudFormation/security_groups_for_rdp/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "security_groups_for_rdp",
+  "queryName": "Security Groups For RDP",
```

Comment:
Change the query name, it should describe the problem:
"queryName": "Security Group Unrestricted Access To RDP",